serverless.yml

```diff
@@ -43,7 +43,7 @@ functions:
       checkpoint: 3
       bucket_name: results-s3-bucket
       identifier_column: responder_id
-      in_file_name: ingest_out.json
+      in_file_name: test_data_mike_1.json
```

Comment:
Ummm... is this meant to be test_data_mike_1 instead of ingest_out?

Reply:
yes, data from ingest is too small. Ian n Paul are aware and happy with this till our input data is better

Reply:
Cool, I'll approve then.